src/wkb.cpp

```diff
@@ -42,6 +40,16 @@ void wkb_read(wkb_buf *wkb, void *dst, size_t n) {
 	wkb->size -= n;
 }
 
+template<typename T>
+inline T wkb_read(wkb_buf *wkb) {
```

Comment:
This new signature of `wkb_read` doesn't provide any performance benefit on its own; I just found it simpler to use.